pkg/backend/httpstate/backend.go

```diff
@@ -787,11 +787,16 @@ func (b *cloudBackend) RenameStack(ctx context.Context, stack backend.Stack, new
 		return err
 	}
 
-	// Transferring stack ownership is available to Pulumi admins, but isn't quite ready
-	// for general use yet.
 	if stackID.Owner != newIdentity.Owner {
+		url := "."
+
+		if consoleURL, err := b.StackConsoleURL(stack.Ref()); err == nil {
```

Comment:
Not a big deal, but maybe it would make sense to extend `StackConsoleURL` function to accept extra paths as additional parameters?

Reply:
As a way to ensure that the path actually exists? Otherwise, not sure how it helps beyond acting like another way to concatenate?

Reply:
Going to merge for now, but happy to come back to this if you feel otherwise. Thanks for the review!